README.md

```diff
@@ -1,2 +1,55 @@
-# qmdataset
-A collection of QM data for training potential functions
+# SPICE: A Dataset for Training Machine Learning Potentials
+
+SPICE (Small-Molecule/Protein Interaction Chemical Energies) is a collection of quantum mechanical data for
+training potential functions.  The emphasis is particularly on simulating drug-like small molecules interacting
+with proteins.  It is designed to achieve the following goals.
+
+- **Cover a wide range of chemical space**.  It includes 15 elements (H, Li, C, N, O, F, Na, Mg, P, S, Cl, K, Ca, Br, I)
+  and a wide range of chemical groups.  It includes charged and polar molecules as well as neutral ones.  It is
+  designed to sample a wide range of both covalent and non-covalent interactions.
+- **Cover a wide range of conformations**.  It includes both low and high energy conformations.  It is
+  designed to sample all regions of configuration space that are likely to be encountered in typical simulations.
+- **Include forces as well as energies**.  Many datasets include only energies, not forces.  SPICE includes
+  forces as well, which enormously increases the information content of the dataset.
+- **Include a variety of other information**.  SPICE includes a variety of other QM results for each sample,
+  such as bond orders, partial charges, and atomic multipoles.  It follows the principle that any result that
+  is easy to compute and potentially useful should be made available.
+- **Use an accuate level of theory**.  Models can never be more accurate than the data they are trained on.
+  SPICE computations are done at the ωB97M-D3BJ/def2-TZVPPD level of theory.
+- **Be a dynamic, growing dataset**.  The dataset should grow with time as new data is generated.  This will
+  allow models trained on it both to improve in accuracy and to expand the range of chemical space they cover.
+  Versioned releases will be created periodically to allow for reproducibility.
+- **Be freely available under a non-restrictive licence**.  All data in the SPICE dataset may be used under the
+  public domain equivalent [CC0 license](https://creativecommons.org/share-your-work/public-domain/cc0/).
+
+SPICE is made up of a collection of subsets.  Each one is designed to provide a particular type of information.
```

Comment:
@peastman : Did you want to include the nice table @pavankum created for his (still pending review) PR to update `README.md` https://github.com/openmm/spice-dataset/pull/20/files ?